website/docs/release_notes.md

```diff
@@ -5,6 +5,91 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.12](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.12)
+
+### Enable passing the `scalafmt` configuration with `--scalafmt-conf` and `--scalafmt-conf-str`
+It is now possible to pass a custom location of the `scalafmt` configuration with the `--scalafmt-conf` option for the
+`fmt` sub-command.
+```text
+$ scala-cli fmt --scalafmt-conf path/to/the/conf/.scalafmt.conf
+```
+You can also pass the configuration straight from the terminal with `--scalafmt-conf-str`.
+```text
+$ scala-cli fmt --scalafmt-conf-str  "version=3.5.5                                   
+runner.dialect=scala213"
+```
+Added in [#1227](https://github.com/VirtusLab/scala-cli/pull/1227) by [wleczny](https://github.com/wleczny)
+
+### Enable turning `--interactive` mode on permanently
+It is now possible to set the `--interactive` mode on by default, so that passing it explicitly isn't necessary.
+
+The next time when you run a command with the `--interactive` option set to on, Scala CLI will suggest to turn it on
+permanently. 
+
+This is recommended for environments where `scala-cli` is used by a human user only. (and not by any automation)
```

Comment:
```suggestion
This is recommended for environments where `scala-cli` is used by a human user only (and not by any automation).
```
I think it looks more natural when not separated from the rest of the sentence

Reply:
fixed